src/components/InlineNotification/InlineNotification.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import CloseIcon from '@mui/icons-material/Close';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import WarningAmberOutlinedIcon from '@mui/icons-material/WarningAmberOutlined';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import CancelOutlinedIcon from '@mui/icons-material/CancelOutlined';
+import { experimental_sx as e_sx, SxProps } from '@mui/system';
+
+import Box, { BoxProps } from '../Box/Box';
+import Typography from '../Typography';
+import IconButton from '../IconButton';
+import Button from '../Button/Button';
+import styled from '../../theme/styled';
+
+export enum NotificationVariant {
+  info = 'info',
+  warning = 'warning',
+  error = 'error'
+}
+
+export interface InlineNotificationProps extends BoxProps {
+  /**
+   * Inline notifications are constrained to `info`, `warning`, or `error`.
+   * Since the default value is `info`, the default component will have styles
+   * related to that variant.
+   * @default `info`
+   */
+  variant?: NotificationVariant;
+  onClose?: (args?: any) => void;
+  /**
+   * 
+   * @default false
+   */
+  startIcon?: boolean;
```

Comment:
Could we maybe use `showStartIcon` here instead? I was initially a little confused because built-in MUI components take a ReactNode for `startIcon`

Reply:
ooh I see; yeah that's a fair point. will change